code.tsx

```diff
@@ -248,6 +259,12 @@ function Widget() {
     }
   );
 
+  useEffect(() => {
+    if (users.length === 0) {
+      updateUsers();
+    }
+  });
```

Comment:
why do you need `useEffect`? Instead let's initialize it in `useSyncedState`?

Reply:
You are not allowed to do that atm